ts/src/pro/coinbase.ts

```diff
@@ -113,7 +113,7 @@ export default class coinbase extends coinbaseRest {
          * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
          */
         if (symbols === undefined) {
-            throw new ArgumentsRequired (this.id + ' watchTickers requires a symbols argument');
+            symbols = this.symbols;
```

Comment:
@ttodua did you test to make sure this works? sometimes you can't subscribe manually to more than X symbols

Reply:
yes ofc, i have tested, and worked well for all symbol, no exceptions, immediately started receiving tickers for different symbols.